psalm-baseline.xml

```diff
@@ -1,10 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="4.23.0@f1fe6ff483bf325c803df9f510d09a03fd796f88">
-    <file src="src/FieldDescription/FieldDescriptionFactory.php">
-        <!-- https://github.com/doctrine/mongodb-odm/pull/2449 -->
-        <ArgumentTypeCoercion occurrences="2">
-            <code>$class</code>
-            <code>$class</code>
-        </ArgumentTypeCoercion>
+<files psalm-version="5.7.7@e028ba46ba0d7f9a78bc3201c251e137383e145f">
+    <file src="tests/Filter/CallbackFilterTest.php">
+        <!-- On purpose, it throws an exception -->
```

Comment:
For exception on purpose, we generally add a psalm-suppress InvalidArgument directly next to the code.